pkg/cloud/gcpprovider.go

```diff
@@ -382,6 +382,10 @@ func (gcp *GCP) GetDisks() ([]byte, error) {
 
 }
 
+func (*GCP) GetOrphanedResources() ([]OrphanedResource, error) {
+	return nil, nil
```

Comment:
These should return `nil, errors.New("not implemented")` so that callers get an indication that this is not yet implemented and don't take `err == nil` to mean "there are results".